src/components/Navbar/Navbar.component.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FunctionComponent } from 'react';
+import city from './../../assets/city-hall.png';
+import home from './../../assets/home.png';
+import logo from './../../assets/logo.png';
+import timetable from './../../assets/timetable.png';
+import './Navbar.style.scss';
+
+const SideBarData = [
+  {
+    title: 'Accueil',
+    path: '/',
+    cName: 'nav-text',
+  },
+  {
+    title: 'Evenement',
+    path: '/evenement',
+    cName: 'nav-text',
+  },
+  {
+    title: 'Vie Etudiante',
+    path: 'vie_etudiante',
+    cName: 'nav-text',
+  },
+];
+
+export const Navbar: FunctionComponent = () => {
+  return (
+    <nav>
+      <img className="logo" src={logo} height="256" width="256"></img>
```

Comment:
height="256" width="256" => put those values in a const that you define at the top of the file, just after file import : const LOGO_IMAGE_SIZE = 256 and then => height={LOGO_IMAGE_SIZE} width={LOGO_IMAGE_SIZE}